tests/unit/src/main/java/org/mule/tck/junit4/AbstractMuleContextTestCase.java

```diff
@@ -308,6 +308,9 @@ protected MuleContext createMuleContext(String contextConfigurationId) throws Ex
   }
 
   protected DefaultMuleConfiguration createMuleConfiguration() {
+    if (getMavenProjectVersionProperty() == null) {
+      return new DefaultMuleConfiguration();
+    }
```

Comment:
@elrodro83 With the changes from https://github.com/mulesoft/mule/pull/11350, the benchmark class started to throw a NullPointerException. To handle the cases without proper version information, is it okay to add this check?

Reply:
@jcgrethe ☝🏼